openmetadata-docs/content/deployment/docker/index.md

```diff
@@ -4,39 +4,174 @@ slug: /deployment/docker
+470cc8149826   openmetadata/server:0.12.2                             "./openmetadata-star…"   45 seconds ago   Up 43 seconds             3306/tcp, 9200/tcp, 9300/tcp, 0.0.0.0:8585-8586->8585-8586/tcp   openmetadata_server
+63578aacbff5   openmetadata/ingestion:0.12.2                          "./ingestion_depende…"   45 seconds ago   Up 43 seconds             0.0.0.0:8080->8080/tcp                                           openmetadata_ingestion
+9f5ee8334f4b   docker.elastic.co/elasticsearch/elasticsearch:7.10.2   "/tini -- /usr/local…"   45 seconds ago   Up 44 seconds             0.0.0.0:9200->9200/tcp, 0.0.0.0:9300->9300/tcp                   openmetadata_elasticsearch
+08947ab3424b   openmetadata/db:0.12.2                                 "/entrypoint.sh mysq…"   45 seconds ago   Up 44 seconds (healthy)   3306/tcp, 33060-33061/tcp                                        openmetadata_mysql
+```
+
+In a few seconds, you should be able to access the OpenMetadata UI at [http://localhost:8585](http://localhost:8585)
+## Port Mapping / Port Forwarding 
 
 We are shipping the OpenMetadata server and UI at `8585`, and the ingestion container (Airflow) at `8080`. You can
-take a look at the official Docker [docs](https://docs.docker.com/compose/compose-file/#ports). As an example, You could
-update the ports to serve Airflow at `1234` with:
+change the port number's according to your requirement. As an example, You could
+update the ports to serve OpenMetadata Server and UI  at port `80`
+
+To achieve this 
+- You just have to update the ports mapping of the openmetadata-server in the `docker-compose.yml` file
 
 ```yaml
 ports:
-  - "1234:8080"
+  - "80:8080"
+```
+- Once the port is updated if there are any containers running remove them first using `docker-compose down` command and then  recreate the containers once again by below command 
+```commandline
+docker compose up --build -d 
+```
+
+## PROD Deployment of OpenMetadata Using Docker
+If you are planning on going to PROD, we recommend to validate below points:
+- MySQL and OpenSearch (ElasticSearch) are available.
+- OpenMetadata-Server require the minimum configuration of 2vCPU and 6Memory (GiB)
+- OpenMetadata-Ingestion require the minimum configuration of 2vCPU and 8Memory (GiB)
+- We also recommend to bind Docker Volumes for data persistence. Learn how to do so [here](/deployment/docker/volumes).
+### Steps for Deploying Ingestion 
+- Download the docker-compose.yml file from the release page [here](https://github.com/open-metadata/OpenMetadata/releases).
+- Update the environment variables below for OpenMetadata-Ingestion Docker Compose backed systems to connect with Database. 
+```
+# MySQL Environment Variables for ingestion service
+DB_HOST: '<YOUR_RDS_HOST_NAME>'
+DB_PORT: '<YOUR_RDS_PORT>'
+AIRFLOW_DB: '<YOUR_RDS_AIRFLOW_DB>'
+AIRFLOW_DB_SCHEME: '<YOUR_RDS_AIRFLOW_DB_SCHEME>'
+DB_USER: '<YOUR_RDS_AIRFLOW_DB_USER>'
+DB_PASSWORD: '<YOUR_RDS_AIRFLOW_DB_PASSWORD>'
```

Comment:
Update the placeholder values to be more generic!

Reply:
Done